packages/next/src/build/webpack/plugins/flight-client-entry-plugin.ts

```diff
@@ -44,20 +45,25 @@ interface Options {
 
 const PLUGIN_NAME = 'FlightClientEntryPlugin'
 
-export type ActionManifest = {
-  [key in 'node' | 'edge']: {
-    [actionId: string]: {
-      workers: {
-        [name: string]: string | number
-      }
-      // Record which layer the action is in (rsc or sc_action), in the specific entry.
-      layer: {
-        [name: string]: string
-      }
+type Actions = {
+  [actionId: string]: {
```

Comment:
Nit: Record<string, {}> might be more readable

Reply:
The thing I like about `{ [actionId: string]: ... }` is that I can have names (i.e. `actionId` in this case) as a note on what each field means.